src/TodoList.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+
+let id = 0
```

Comment:
Nitpick (doesn't require changes): 
This `let id = 0` variable can be moved into your `buildList` function since it only needs to be accessed/modified by this function.

Reply:
Yeah, I see what you mean. You're definitely right. The reason I left it out of the function was I'm assuming that when I start adding items to the list later, I'll be incrementing and using the same id variable to avoid duplicate ids. But, I'm not even sure though if that's how I will be implementing it yet, and as it stands right now, it definitely doesn't make sense to have it that way. I was just getting ahead of myself, haha. I'll be more mindful of that in future submissions.